src/yunohost/domain.py

```diff
@@ -160,12 +154,14 @@ def domain_add(auth, domain, dyndns=False):
             with open('/etc/yunohost/installed', 'r') as f:
                 service_regen_conf(names=[
                     'nginx', 'metronome', 'dnsmasq', 'rmilter'])
-                os.system('yunohost app ssowatconf > /dev/null 2>&1')
+                assert os.system('yunohost app ssowatconf > /dev/null 2>&1') == 0, "SSOwat conf regen failed"
```

Comment:
It's really ugly and should be replaced by `app_ssowatconf` method call directly......